docs/DesignDocs/PluggableAllocators.md

```diff
@@ -0,0 +1,142 @@
+Pluggable Allocators
+=====
+
+Open Enclave provides a default memory allocator for enclaves.
+This default allocator, [dlmalloc (Doug Lea Malloc)](http://gee.cs.oswego.edu/dl/html/malloc.html), is not performant in multi-thread applications.
+
+There are various high-performance memory allocators that try to optimize, among other things, multi-thread scenarios:
+- [The GNU Allocator](https://www.gnu.org/software/libc/manual/html_node/The-GNU-Allocator.html) which is based on `pthreads malloc`, which is in turn based on `dlmalloc`.
+- [Thread-Caching Malloc (tcmalloc)](https://gperftools.github.io/gperftools/tcmalloc.html) developed by Google.
```

Comment:
Why is who it's developed by relevant?  Line 8 didn't say who that one was developed by.  Recommend deleting the developed by in lines 9 and 10.  I can imagine that which LICENSE it's under might be relevant, but the developer should be immaterial here.

Reply:
Thanks for the suggestions. Auto incorporated.